### DISCUSSION TITLE
BPF to only INCLUDE hosts

### DISCUSSION THREAD

**Discussion by dootdootbeepbeep** [Security-Onion-Solutions/securityonion#3925] [Unsupported Versions] [answered]
We are trying to monitor only traffic going to or from certain hosts. Using BPF do I have to NOT everything else or can I just allow from the desired hosts?

For example I want to monitor traffic coming to or going from 10.10.10.10, 10.5.3.2 so on and so on.
Can I just have a bpf to allow those or must it be not everything else?

**Comment by weslambert** [comment, marked as answer]
In this case, you would not use negation.

For example:

`host 192.168.1.10`  would filter out anything but traffic to/from 192.168.1.10
